testsuite/cluster-test/src/cluster_swarm/mod.rs

```diff
@@ -34,7 +34,7 @@ pub trait ClusterSwarm {
     ) -> Result<Instance>;
 
     /// Deletes a node from the ClusterSwarm
-    async fn delete_node(&self, instance: &Instance) -> Result<()>;
```

Comment:
You just made this opposite change :D

Reply:
I know)
But now I wanted to get rid of usage of `Instance::k8s_node_name()`